sjb/config/test_cases/test_branch_ansible_service_broker.yml

```diff
@@ -0,0 +1,93 @@
+---
+parent: 'common/test_cases/origin_built_release.yml'
```

Comment:
You should consider making the parent test_cases/test_branch_origin_service_catalog.yml and remove all the origin/service-catalog building steps. The alternative is that you have to maintain the catalog build steps yourself, but it's not a clear decision since you'd be able to skip running catalog tests with the way things currently are.